docs/en/academic/custom-templates.md

```diff
@@ -1,83 +1,134 @@
 
 ![Create TeX-Files by appending the appropriate extension](../img/create_tex_file.png)
 
-If you create a new file, but **provide `.tex` as the file extension**, Zettlr will not create a Markdown file but an actual LaTeX file! This file will be indicated with a small `TeX` indicator in the file list (if file meta is turned on) and can be edited from within Zettlr. Hooray!
+First, create a new file and **give it a `.tex` file extension**.  A small `TeX` indicator will appear beneath the filename in the file list.
 
+<!-- TO CHECK: If I save a file with a file extension of .tex, then Zettlr will show the small 'TeX' indicator. However, if you rename an existing file and give it a .tex file extension, Zettlr will now show a small 'TeX' indicator. I would have expected the 'TeX' indicator to be show in both circumstances. 
+
+A new figure is needed; however I am hesistant to replace the current one as my Zettlr interface looks different (which is to be expected given I am using 2.0.0.0-beta; however, I don't have the + symbol to add files...so I thought it best to wait to hear back on the the above comment before replacing the figure. -->
 ![Zettlr with a TeX file open](../img/zettlt_tex_file.png)
 
-Zettlr will automatically detect if it's a LaTeX file and even switch the code highlighting from Markdown to LaTeX to help you write the file.
+Zettlr will automatically switch code highlighting from Markdown to LaTeX.
 
 ## Necessary Contents
 
-All of your templates need two specific things. First, as we have already said, you must include the `$body$`-variable so that your processed file contents end up inside the file and are not dumped into oblivion.
+While you could omit all Zettlr-specific variables, there is one Pandoc variable that needs to be present at all times:
+
+```
+$body$
+```
+
+Pandoc will replace this variable with the parsed contents of your Markdown file(s). If you leave it out, your content will not appear in the output file.
+
+> The default Zettlr template does not include many of the variables from the default Pandoc template. However, you are free to re-introduce some or all of them. This ability to use a combination of Pandoc and Zettlr variables makes template files truly powerful!
 
-The second part which all of your templates need is the boilerplate required to make your file compile. Especially LaTeX templates require a large amount of directives to work at all. Always remember that Pandoc will make use of LaTeX commands internally which must be defined in the head section of the template.
+## Hacking Your Templates!
+
+<!-- TO CHECK: Is still section relevant in V2? I could not find where in preferences you can amend the default Pandoc command used.  Has this functionality been removed from V2 or has it be moved? If it has been moved, then this section should be re-written by someone who knows how V2 is intented to work with this notion of hacking templates.-->
```

Comment:
Nop, since we don't have a Pandoc command anymore, this is not relevant anymore. With regard to running custom logic, there's an issue open for this, but it's not possible right now, so we can safely remove that.

Reply:
Ok. I will delete.